descriptors/ModuleDescriptor-template.json

```diff
@@ -90,7 +90,11 @@
         }, {
           "methods": ["POST"],
           "pathPattern": "/holdings-storage/holdings",
-          "permissionsRequired": ["inventory-storage.holdings.item.post"]
+          "permissionsRequired": ["inventory-storage.holdings.item.post"],
+          "modulePermissions": [
+            "user-tenants.collection.get",
+            "consortia.sharing-instances.item.post"
```

Comment:
I assume, there won't be any problems in case this permission does not exist, right?

Reply:
yes